powershap/shap_wrappers/shap_explainer.py

```diff
@@ -171,11 +171,15 @@ def explain(
             Shap_values = np.abs(Shap_values)
 
             if len(np.shape(Shap_values)) > 2:
-                # Shap_values = np.max(Shap_values, axis=0)
-                Shap_values = np.max(Shap_values, axis=0).T
+                # SHAPE: (n_samples, n_features, n_outputs)
```

Comment:
instead of `n_outputs` maybe `n_classes`?

Reply:
voor multi-regression is het dan n_outputs